docker/Dockerfile

```diff
@@ -6,34 +6,37 @@ ENV TEMPO_VERSION="2.4.1"
 ENV LOKI_VERSION="2.9.6"
 ENV OPENTELEMETRY_COLLECTOR_VERSION="0.97.0"
 
+ARG TARGETARCH
```

Comment:
can you make `amd64` the default, similar to `RELEASE`?

Reply:
Hi @zeitlinger,

thanks for your comment. It shouldn't be necessary: The `TARGETARCH` is used as a kind of magic variable in Docker when building images. If the build is started with a Docker running on an `amd64`, then `TARGETARCH` would set to `amd64` automatically.

But if there's a reason (pipeline, etc) I could do this. :)

Cheers

Reply:
ah, didn't know!

Can you just add a comment explaining that's it's filled automatically?

Reply:
Have added some remarks regarding the usage of that variable.